### ISSUE TITLE
window screen location issue with multiple screens

### ISSUE THREAD

**Issue by Overdrivr** [zeit/hyper#1608]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **Windows 7 enterprise SP1** 
- **Hyper.app version**: 1.2.1 
- **The issue is reproducible in vanilla Hyper.app**: yes 

## Issue

Required hardware configuration is a laptop connected to a dock with extra screen.

Steps to reproduce the issue:
1. Open hyper and move its window to the extra screen
2. Close hyper
3. Disconnect laptop from dock.
4. Open hyper again
5. Hyper is still created at the former location (that now points outside my laptop screen), and there is no way to bring it back except by restarting the laptop entirely.

Basically, the issue comes from the very poor additionnal screen management in Windows, but most other programs I use like Atom for instance handle this, probably by resetting the window location on startup.

**Comment by ppot** [comment]
@Overdrivr Yes, this is an issue since we memorize the screen position on quit. But if an extended display is disconnected. It will open at the recorded position. You can temporaly overide this by deleting the `electron-config` cache.

**Comment by a-type** [comment]
For those like me that went hunting around, the electron-config cache is at `AppData/Roaming/Hyper/config.json`. Deleting it does bring back the window.

**Comment by MartyGentillon** [comment]
Alternately, you should be able to move the window around using the normal windows window moving keys (windows+left, windows+right, and windows+up).  That should get it back on screen pretty easily.

**Comment by a-type** [comment]
@MartyGentillon tried this, but Hyper doesn't seem to respond. Also tried moving it to a new virtual desktop.

**Comment by MaxBackman** [comment]
What worked for me was actually adjusting the "scaling" on my laptop display from 125% (preset) to 100%, something about this made Hyper open off-screen on the external monitor.

**Comment by chabou** [comment]
Will be fixed by: https://github.com/zeit/hyper/pull/2459
